apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.thumbnail.presenter.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+    IDetailQueryResult,
+    ITranscribedAudioViewModel,
+    ResourceType,
+} from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import { Link } from 'react-router-dom';
+import { routes } from '../../../app/routes/routes';
+import './transcribed-audio-detail.thumbnail.presenter.css';
+
+export const TranscribedAudioDetailThumbnailPresenter = ({
+    data: transcribedAudioItem,
+}: IDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => {
+    const location = `/${routes.resources
+        .ofType(ResourceType.transcribedAudio)
+        .detail(transcribedAudioItem.id)}`;
+
+    const runTime = Math.round(transcribedAudioItem.length / 1000);
```

Comment:
You need a util that does this, e.g. `convertSecondsToMilliseconds`. Eventually, we may want a math lib, for now make it a util on the front-end.

Reply:
Just want to be sure I'm on the same page with you about this over the different comments.  A few different strategies flying about here.

Reply:
Fixed this with the convert utility